patches/chromium/don_t_run_pcscan_notifythreadcreated_if_pcscan_is_disabled.patch

```diff
@@ -35,3 +55,33 @@ index dbf96f68f9d8ca74616af10c236ebe24984b2b5e..e11237ad35743e9cc724f4974662a5ed
    internal::PCScan::NotifyThreadDestroyed();
  #endif
  
+diff --git a/base/threading/platform_thread_win.cc b/base/threading/platform_thread_win.cc
+index 86e240b231d3236aea1b0343e0e6763282ff70c9..0d85d2acb6652a580f79a2c4ca2277562dd5156b 100644
+--- a/base/threading/platform_thread_win.cc
++++ b/base/threading/platform_thread_win.cc
+@@ -28,6 +28,7 @@
+ #include <windows.h>
+ 
+ #if BUILDFLAG(USE_PARTITION_ALLOC_AS_MALLOC)
++#include "base/allocator/partition_allocator/partition_alloc_config.h"
+ #include "base/allocator/partition_allocator/starscan/pcscan.h"
+ #include "base/allocator/partition_allocator/starscan/stack/stack.h"
+ #endif
+@@ -104,7 +105,7 @@ DWORD __stdcall ThreadFunc(void* params) {
+                                 FALSE,
+                                 DUPLICATE_SAME_ACCESS);
+ 
+-#if BUILDFLAG(USE_PARTITION_ALLOC_AS_MALLOC)
++#if BUILDFLAG(USE_PARTITION_ALLOC_AS_MALLOC) && defined(PA_ALLOW_PCSCAN)
+   internal::PCScan::NotifyThreadCreated(internal::GetStackPointer());
```

Comment:
Should this perhaps also check the feature `kPartitionAllocPCScan`?

Reply:
Hmm not sure... I upstreamed the patch here:
https://chromium-review.googlesource.com/c/chromium/src/+/2965109